cmd/cli/mesh_upgrade.go

```diff
@@ -23,18 +23,15 @@ const (
 const upgradeDesc = `
 This command upgrades an OSM control plane configuration by upgrading the
```

Comment:
Could you drop this change from this PR? Don't see it being necessary here.

Reply:
reverted